Client/src/Components/Google.jsx

```diff
@@ -0,0 +1,21 @@
+function Google(){

```

Comment:
delete this component, use the icon as an image instead

Reply:
but i can't get this icon image from figma
it is available as svg links
can i use from google images

Reply:
yes you can use google images